### ISSUE TITLE
Some Emoji are not displayed correctly

### ISSUE THREAD

**Issue by dbrgn** [threema-ch/threema-web#54]
https://twitter.com/NerdingByDoing/status/832225568322097152

Sending the sun emoji from Android results in the following span:

```javascript
> emojione.unicodeToImage('☀')
"<span class="e1 e1-2600" title=":sunny:">☀</span>"
```

On the other hand, sending the same emoji from iOS results in this span:

```javascript
> emojione.unicodeToImage('☀️')
"<span class="e1 e1-2600-fe0f" title=":sunny:">☀️</span>"
```

The codepoint U+FE0F is the emoji variation selector: https://unicode-table.com/de/FE0F/. On the other hand, U+FE0E would be the text variation selector.

The sprites CSS generated by the emojione library does not contain the 2600-fe0f variant.

Unfortunately, it looks like emojione currently does not support these variants: https://github.com/Ranks/emojione/issues/358

**Comment by corrupt** [comment]
We have found a number of other emoji where this happens. And it's not like emojione didn't support variations at all. Maybe this is an upstream bug.

Here's another example of a quick test from earlier:
![_20170216_202253_copy](https://cloud.githubusercontent.com/assets/365169/23037264/0636e49e-f486-11e6-9f79-7285915c55b3.JPG)
![screenshot_20170216_202205](https://cloud.githubusercontent.com/assets/365169/23037265/063b9a8e-f486-11e6-8834-7b4771f15da7.png)

The dark image is from Threema's Android app, obviously. The sender uses an up-to-date iPhone. Notice the part in the middle with the two girls and two boys. It is replaced by a single cop emoji as the code is not parsed correctly. Threema Web tries to combine the emoji and their selectors into one CSS class.

```html
<span class="e1 e1-1f469-200d-1f469-200d-1f466-200d-1f466" title=":family_wwbb:">👩‍👩‍👦‍👦</span>
```

**Comment by dbrgn** [comment]
This should get fixed with the new emojione release v3:

http://emojione.com/

**Comment by dbrgn** [comment]
To see the progress of the fix, please refer to #201.
